### ISSUE TITLE
unknown command throws error

### ISSUE THREAD

**Issue by plexigras** [cacjs/cac#18]
The most bare bone example I could come up with

```
$ node index someUnknownCommand
```

```js
// index.js
require('cac')().parse()
```

```
/home/superuser/git/cac-test/node_modules/cac/dist/cac.js:171
    const commandText = chalk.magenta(this.opts.displayCommands ? '<command> ' : `${this.command.command.name} `);
                                                                                                 ^

TypeError: Cannot read property 'command' of undefined
    at Help.getHelp (/home/superuser/git/cac-test/node_modules/cac/dist/cac.js:171:98)
    at Help.output (/home/superuser/git/cac-test/node_modules/cac/dist/cac.js:207:31)
    at Cac.showHelp (/home/superuser/git/cac-test/node_modules/cac/dist/cac.js:378:10)
    at Object.<anonymous> (/home/superuser/git/cac-test/index.js:3:5)
    at Module._compile (module.js:662:30)
    at Object.Module._extensions..js (module.js:673:10)
    at Module.load (module.js:575:32)
    at tryModuleLoad (module.js:515:12)
    at Function.Module._load (module.js:507:3)
    at Function.Module.runMain (module.js:703:10)
```

The only way I could get rid of this error was by adding a matching command such as `*` or `someUnknownCommand`.

Tested in version `4.4.2`, `4.4.1`, `4.4.0` and `4.3.7`.

**Comment by egoist** [comment]
I guess we need to make `commandText` an empty string when there's no matched command:

https://github.com/cacjs/cac/blob/c989e431b1349da0406e7d20da5cca32ca93ddd3/src/Help.js#L26-L28

**Comment by plexigras** [comment]
i think i found an easier way to fix this

**Comment by egoist** [comment]
Should be fixed in 4.4.3
